app/src/main/java/com/mstudio/superheroarch/Character.kt

```diff
@@ -1,15 +1,34 @@
 package com.mstudio.superheroarch
 
+import androidx.room.Entity
+import androidx.room.PrimaryKey
 import java.io.Serializable
 
+@Entity(tableName = "characters")
 data class Character(
-    val name: String,
+    @PrimaryKey val name: String,
```

Comment:
a name cannot be a primarykey, you don't know if there are several characters with same name.